diff --git a/taox11/tao/x11/system_exception.h b/taox11/tao/x11/system_exception.h
index 9a5349f..fd83ca7 100644
--- a/taox11/tao/x11/system_exception.h
+++ b/taox11/tao/x11/system_exception.h
@@ -92,6 +92,7 @@ namespace TAOX11_NAMESPACE
                        uint32_t code,
                        CORBA::CompletionStatus completed);
 
+    public:
       void _info (std::ostream& strm) const override;
 
     private:
